topics/assembly/tutorials/largegenome/tutorial.md

```diff
@@ -0,0 +1,587 @@
+  * Medaka polished assembly => Fasta statistics
+* Workflow steps for Part 3, Polish with short reads: using Racon
+  * Short reads and Medaka polished assembly =>Racon polish (subworkflow):
+  * minimap2: short reads (R1 only) are mapped to the assembly => overlaps.paf. Minimap2 setting is for short reads.
+  * overlaps + short reads + assembly => Racon => polished assembly 1
+  * using polished assembly 1 as input; repeat minimap2 + racon => polished assembly 2
+  * Racon short-read polished assembly => Fasta statistics
+* Options:
+  * Change settings for Racon long read polishing if using PacBio reads:  The default profile setting for Racon long read polishing: minimap2 read mapping is "Oxford Nanopore read to reference mapping", which is specified as an input parameter to the whole Assembly polishing workflow, as text: map-ont. If you are not using nanopore reads and/or need a different setting, change this input. To see the other available settings, open the minimap2 tool, find "Select a profile of preset options", and click on the drop down menu. For each described option, there is a short text in brackets at the end (e.g. map-pb). This is the text to enter into the assembly polishing workflow at runtime instead of the default (map-ont).
+* Other options: change the number of polishes (in Racon and/or Medaka). There are ways to assess how much improvement in assembly quality has occurred per polishing round (for example, the number of corrections made; the change in Busco score - see section "Genome quality assessment" for more on Busco).
+* Option: change polishing settings for any of these tools. Note: for Racon - these will have to be changed within those subworkflows first. Then, in the main workflow, update the subworkflows, and re-save. 
+>
+{: .hands_on}
+
+*Polishing results*
+
+The polished assembly is the final Racon file. 
+
+Look at the Fasta Statistics output files for the status of the assemblies after each polishing tool. From these, we can see that some polishes decrease or increase the size of the assembly. The final size is approximately 300,000 bp shorter than the original Flye assembly. (These numbers may vary slightly even if the same input data is used). 
+
+# Genome quality assessment
+
+The polished genome is in 145 contigs with a total length of ~ 9.6 million base pairs.  We have some idea of how these contigs may be joined (or not) from the Bandage assembly graph. 
+
+> ### {% icon question %} Questions
+>
+> 1. How good is the assembly?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1.  One measure is the N50, a number that indicates how large the contigs are (although, have the contigs been joined correctly)? Another measure is to see if expected gene sequences are found in the assembly, using a tool called BUSCO. For a discussion on these and other methods, see {% cite Wang2020%}.
+> >
+> {: .solution}
+{: .question}
```

Comment:
I would move that question to the end of the section.

Reply:
I think this is ok for now.